config/file.go

```diff
@@ -160,6 +164,71 @@ func (fs *FileStore) Save() error {
 	return fs.persist(fs.config)
 }
 
+// GetFile fetches the contents of a previously persisted configuration file.
+func (fs *FileStore) GetFile(name string) ([]byte, error) {
+	resolvedPath, err := resolveConfigFilePath(name)
```

Comment:
Reflecting on the original implementation, I don't think this actually has to resolve the file: it should probably just `filepath.Join("config", filepath.Base(name))`. I'll investigate this further.

Reply:
I ended up extracting out a number of commits from this PR into separate reviews, and did indeed converge on the simpler file lookup.